server/entity/experience.go

```diff
@@ -108,13 +116,13 @@ func progressFromExperience(experience int) (level int, progress float64) {
 }
 
 // experienceForLevels calculates the amount of experience needed in total to reach a certain level.
-func experienceForLevels(level int) int {
+func experienceForLevels(level int) float64 {
```

Comment:
Would probably go back to returning an int here too. The less we return floats in these functions the better I think.

Reply:
Might lose some experience points by truncating here

Reply:
Technically though experience can only be full numbers?